base/base.gyp

```diff
@@ -1,63 +0,0 @@
-# Copyright (c) 2009 The Chromium Authors. All rights reserved.
-# Use of this source code is governed by a BSD-style license that can be
```

Comment:
While we're not confident that the `bazel` branch is on-par, changes that land on `master` need to be cherry-picked here. I wasn't thinking about merging this anytime soon yet, it's still a WIP. Running two branches seems less work compared to attempting to do it on a single branch (though not impossible of course), because of the current grpc version on master being non-trivial to port. There's some bloat from the updated/leaned up chromium dependencies; it would be a bit more work to get the bazel branch working with the current version instead of updating the dependency right away.